flake.nix

```diff
@@ -60,7 +64,15 @@
           packages.${name} = buildRustPackage rec {
             inherit name;
             pname = name;
-            src = nixpkgs.lib.cleanSource ./.;
+            src = nix-filter.lib {
+              root = ./.;
+              include = [
+                (nix-filter.lib.matchExt "rs")
+                ./src/ragenix/agenix.schema.json
```

Comment:
I guess I just would have included `src` altogether. Isn't that possible?

Reply:
there is `inDirectory` but I would recommend against the use of `nix-filter` in it's curring state, It seems to cause more trouble than it solves.

In order to really do the source filtering one wants, one has to run the debug traces on all file paths. Otherwise it's just too counter-intuitive when 

```
                (nix-filter.lib.matchExt "rs")
                ./src/ragenix/agenix.schema.json
```

_also match `./src/ragenix/mod.rs` but _not_ `./test/*.rs`. And

Reply:
I'm going to revert that commit. We'd have to look for a better importer, I also raised the issue upstream.